package.json

```diff
@@ -36,6 +36,7 @@
     "if-env": "^1.0.4",
     "jquery": "^3.5.1",
     "moment": "^2.26.0",
+    "mongodb": "^3.5.8",
```

Comment:
Why are we installing mongodb? All of our interfacing with mongoDB should be managed by mongoose.